src/sentry/monitors/clock_dispatch.py

```diff
@@ -70,6 +78,50 @@ def _dispatch_tick(ts: datetime):
     _clock_tick_producer.produce(ArroyoTopic(topic), payload)
 
 
+def _make_reference_ts(ts: datetime):
+    """
+    Produce a timestamp number with the seconds and microsecond removed
+    """
+    return int(ts.replace(second=0, microsecond=0).timestamp())
+
+
+def update_check_in_volume(ts: datetime):
```

Comment:
I think we can delete this and just call `bulk_update_check_in_volume([ts])` where we use it

Reply:
Sounds good

Reply:
Will follow up. Want to get the deploy rolling on this.

Reply:
https://github.com/getsentry/sentry/pull/79558